index.html

```diff
@@ -1,11 +1,147 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang = "en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta charset = "UTF-8">
+  <meta name = "viewport" content = "width=device-width, initial-scale=1.0">
+  <link rel = "stylesheet" href = "style.css">
   <title>Challenge 1</title>
 </head>
+
 <body>
-  <h1>Challenge 1</h1>
+
+  <h1>COMPUMUNDO HIPERMEGA RED</h1>
+
+  <main>
+
+    <div class = "Heading-Line">
+      <hr class = "Heading-hr">
+    </div>
+
+    <section class = "Main-Charter">
+      <div class = "HeadingImageBox">
+        <img width = "500" src = "https://www.notebookcheck.org/fileadmin/Notebooks/News/_nc3/Zephyrus_Duo93.jpg" alt = "Asus Zephyrus">
+      </div>
+
+      <div class = "HeadingBox">
+        <div class = "FeaturedCircle">
+          <img width = "50" src = "https://www.pnguniverse.com/wp-content/uploads/2020/10/Homero-Simpson-1024x490.png" alt = "Homero Simpson" id = "SL">
+          <p><strong>Featured by</strong></p>
+        </div>
+
+        <div class = "BusinessBox">
+          <div class = "Circle">
+            <p>Business</p>
+          </div>
+          <div class = "Date">
+            <p>August 19, 2023</p>
+          </div>
+        </div>
+
+        <h2>COMPUMUNDO HIPERMEGA RED presenta</h2>
+        <p>"Esta industria avanza tan rápido que es difícil saber"</p>
+
+        <div class = "HeadingBoxBy">
+          <div class = "HeadingInnerBox">
+            <img width = "50" src = "https://www.dibujos-faciles.com/wp-content/uploads/2022/12/dibujo-de-Homer-Simpson-paso7-1.jpg" alt = "Homero Simpson">
+          </div>
+
+          <div class = "HeadingInnerBox2">
+            <p>by <strong>Homero Simpson</strong></p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class = "hr-Charter">
+      <div class = "Separator-Box">
+        <div class = "Separator-Text">
+          <p><strong>Trending now</strong></p>
+        </div>
+
+        <div class = "Separator-Line">
+          <hr class = "styled-hr">
+        </div>
+      </div>
+    </section>
+
+    <section class = "Sub-Charter">
+      <div class = "C1">
```

Comment:
C1 no es una clase muy diciente, qué tal algo como `card` ?
```suggestion
      <div class = "card">
```

Reply:
Nombre cambiado